### DISCUSSION TITLE
Downloading only Movies in Autobrr

### DISCUSSION THREAD

**Discussion by Kaz242** [autobrr/autobrr#1379] [General]
Hi, I'm new to using autobrr and I'm trying to download only freeleech Movies and no TV shows.  Currently it's downloading both, I can't seem to work this out.  

Can someone please help me with this, do I go into Categories in the advanced tab and type Movies on there or what should I add if I only want to download freeleech Movies and no TV shows?

**Comment by zze0s** [comment]
Hi! You need to use categories with **Match Category** to control that. Check our docs here where we have collected some of the announced categories for various indexers. https://autobrr.com/filters/categories

**Reply by Kaz242** [reply]
Thank you!

**Reply by Kaz242** [reply]
Hi, can I just check when I'm adding the category's, do I add a space after the comma for it to work or should it be without the space?
On the category I have added this as Movies :: Bluray,Movies :: WEBRip or should there be with a space after the comma like 
Movies :: Bluray, Movies :: WEBRip

Also, when specifying the years on the TV/Movies tab, if I want to download stuff between the years 1980 to 2024, nothing before 1980.  How should I add this as? I tried adding it as 1980-2024 but I don't think it worked.

Thanks

**Reply by zze0s** [reply]
If you want all movie categories you can do `*movies*` and it will use all of them. Space or no space after comma does not matter.

As for year then yes, `1980-2024` should work. Just beware that not all releases have year in the name.

**Reply by Kaz242** [reply]
Thanks
